pkg/gen/genbzl/main.go

```diff
@@ -0,0 +1,131 @@
+// Command genbzl is used to generate bazel files which then get imported by
```

Comment:
Nit: the other Go binaries we use for generating Bazel stuff are in `pkg/cmd` (e.g.: `mirror`, `generate-staticcheck`, `generate-test-suites`). This one should live in the same place IMO.

Reply:
I have noticed that, but I think, honestly, it's a mistake. We've long had binaries all over the place. The `cmd` directory, as far as I'm concerned, is for binaries which we might expect somebody to want to actually run as a binary. Maybe that's controversial. I'd honestly be more inclined to move all of those to some common subdirectory which makes it more obvious what they are there for.

Reply:
👍 Makes sense to me.